core/src/apps/base.py

```diff
@@ -24,6 +24,9 @@
     )
 
 
+_SCREENSAVER_IS_ON = False
```

Comment:
It will, but the timeout will trigger immediately and re-set the right value before a real homescreen has the chance to start up.
I suspect there's a race condition hiding there somewhere so we still need a better solution for the future, but for now this is fine.